samples/bluetooth/fast_pair/locator_tag/sample.yaml

```diff
@@ -12,8 +12,6 @@ tests:
       - nrf5340dk/nrf5340/cpuapp
       - nrf5340dk/nrf5340/cpuapp/ns
       - nrf54l15dk/nrf54l05/cpuapp
-      - nrf54l15dk/nrf54l10/cpuapp
```

Comment:
We used to test on `nrf54l15dk/nrf54l10/cpuapp` and skip `nrf54l15dk/nrf54l05/cpuapp` (in `sample.bluetooth.fast_pair.locator_tag` test scenario)